Scala2-Functional-Programming/Week6/forcomp/src/main/scala/forcomp/Anagrams.scala

```diff
@@ -134,7 +160,26 @@ object Anagrams extends AnagramsInterface {
    *
    *  Note: There is only one anagram of an empty sentence.
    */
-  def sentenceAnagrams(sentence: Sentence): List[Sentence] = ???
+  def sentenceAnagrams(sentence: Sentence): List[Sentence] = {
+    val lettersOcc: Occurrences = sentenceOccurrences(sentence)
+
+    def helper(current: Sentence, leftOccurrence: Occurrences): List[Sentence]  = {
```

Comment:
Not a big fan of function named `helper`. 
Those names are usually given to functions that help with more than one thing.
Please rename or make sure that function really does one single thing

Reply:
change the name, I would like to move on to other items, if you have a different solution I will gladly see it